src/renderer/lens-app.tsx

```diff
@@ -52,8 +50,6 @@ export class LensApp extends React.Component {
       <Router history={history}>
         <ErrorBoundary>
           <Switch>
-            {UserStore.getInstance().isNewVersion && <Route component={WhatsNew}/>}
-            <Route component={WhatsNew} {...whatsNewRoute}/>
```

Comment:
Note that we might want to merge `ClusterManager` to `LensApp` now that `LensApp` does not really have any custom routes. But it's out-of-scope for this PR.